### PR TITLE
Updates for log 0.4.

### LINKED ISSUE #8: Update to log 0.4.x
log 0.4 is now out.

### PR DESCRIPTION
**Pull request by remexre**
I ran `cargo test` but otherwise haven't tested extensively; this *should*  work, though. I also ran `rustfmt` against it, which is what changed the whitespace in the tests. fixed #8

### CODE REVIEW

**Comment by remexre** [comment]
Looks like this might also change the earliest supported version to be 1.16.0 (at least, that's the version that log itself is testing with).

**Comment by cardoe** [comment]
Thanks for doing this @remexre. I'll work on getting this landed and a new version out.

**Comment by cardoe** [comment]
The minimum supported version is actually 1.20. Due to https://github.com/rust-lang-nursery/bitflags

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # ChangeLog
 
+## 0.3.0
+
+### Changed
+
+- Now depends on clap 0.4.1.
```
Comment (cardoe) [inline CHANGELOG.md]: log not clap

File: Cargo.toml [hunk 2]
```diff
@@ -10,11 +10,11 @@ name = "stderrlog"
 readme = "README.md"
 changelog = "CHANGELOG.md"
 repository = "https://github.com/cardoe/stderrlog-rs"
-version = "0.2.4-pre"
+version = "0.3.0"
```
Comment (cardoe) [inline Cargo.toml]: I use `cargo-release` for this so you don't need to mess with it. This will actually become 0.4.0

**Comment by remexre** [comment]
Sorry for disappearing for a bit; this all should(?) work?

**Comment by cardoe** [comment]
Thanks for doing this @remexre. Which version of `rustfmt` did you use? It appears a decent chunk is style re-formating.

**Comment by remexre** [comment]
Reformatted again with `rustfmt 0.3.4-nightly (6714a44 2017-12-23)`.

**Comment by cardoe** [comment]
Well I'm not actually looking to have the code reformatted with your changes. That just adds to the churn and takes away from the actual meat of the changes. I was curious what you used so that I could do the inverse and just see what you actually changed.

**Comment by remexre** [comment]
Oh, huh, okay, I'll try to unformat it.

**Comment by cardoe** [comment]
I can format code the same way first. I personally still use rustfmt 0.8.6

**Comment by remexre** [comment]
Hm, it looks like even with a copy of rustfmt I just built off the 0.8.6 release, there's lots of formatting differences; it looks like my rustfmt isn't indenting closures like [this](https://github.com/cardoe/stderrlog-rs/blob/master/tests/utils/mod.rs#L37-L42)?

**Comment by Mange** [comment]
Any chance of getting this ball rolling again? :-)

**Comment by remexre** [comment]
I've been using my fork in small-scale production for about a month; I'm not sure if anything other than sorting out the rustfmt issues needs to be done here. I'm

**Comment by cardoe** [comment]
merged in 0f95f1eddc268dec

**Comment by cardoe** [comment]
For those interested this has now been released as 0.4.0 and available on crates.io I waited a few days hoping for some feedback on the project at https://internals.rust-lang.org/t/looking-for-an-eval-of-stderrlog/7391 but didn't get any so I went ahead with the release.

**Pull request closed without merging**